client/src/app/_sandbox/dynamic-components/type-ahead/type-ahead.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-type-ahead',
+  templateUrl: './type-ahead.component.html',
+  styleUrls: ['./type-ahead.component.scss']
+})
+export class TypeAheadComponent implements OnInit {
```

Comment:
Please bshare-typeahead as the kebab case name
For the component name you can let it as : BShareTypeahead

Reply:
fixed